examples/machine-learning/a3-ultragpu-8g/nccl-tests/README

```diff
@@ -0,0 +1,89 @@
+The examples in this directory are used to show how enroot + pyxis can be used
+to launch containerized workloads via Slurm.
+
+Contents:
+
+* `import_container.sh`: Uses enroot to create a squashfs container image.
+* `build-nccl-tests.sh`: A Slurm batch script for building the nccl-tests.
+* `run-nccl-tests.sh`: A Slurm batch script for running the nccl-tests
+  `all_reduce_perf` benchmark.
+
+# Running NCCL-Tests via Enroot/Pyxis
+
+In general the workflow to deploy GPUDirect-RDMA-enabled workloads via enroot-pyxis is
+the following:
+
+	1. Convert your container into a squashfs based container image
+	2. Set required environment variables
+	3. Run your application workload
+
+## TLDR
+
+For an end-to-end example, copy the `build-nccl-tests.sh` and
+`run-nccl-tests.sh` to your login node.
+
+And run the following:
+
+	enroot import docker://nvcr.io#nvidia/pytorch:24.09-py3 # takes ~10 minutes
```

Comment:
If the files aren't combined, shouldn't they run `bash import_pytorch_containers.sh`

Reply:
Combined